### PR TITLE
Portfolio page Mobile Version Skeleton Card and About Section 

### PR DESCRIPTION
**Pull request by henrycode460**
Mobile Version Skeleton of a page - Card and About Section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+        <!-- single card start here -->
+        
+        <div class="card fouth-card"> 
+            <img src="cardFourImage.svg" alt="" class="card-img">
+
+        <h2 class="project-title">Multi-Post
+            Stories</h2>
+
+        <ul class="workInfo">
+            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
```
Comment (MeqdamAlqudah) [inline index.html:157]: - [ ] Please fix the aligning of each project use flexbox to align each one of them 
image

- Your design doesn't look like [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design)

| Your design | Figma design |
| ------------- | --------------- |
| image | image |
| look at this weird doubling in the card please fix it | |

File: index.html [hunk 2]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="tag-btn-container">
+            <li class="tag-btn"> <button type="button"  class="btn-small">html</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">css</button> </li>
+            <li class="tag-btn"> <button type="button"  class="btn-small">javascript</button></li>
+        </ul>
+
+        <button type="button"  class="btnNormal">See Project</button>
```
Comment (MeqdamAlqudah) [inline index.html:172]: - [ ] Please fix margins (top, bottom, left, right) defined in the design

| Your design | Figma design |
| ------------- | --------------- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+            
+        </div>
+
+        <!-- card ends here -->
+
+
+        <!-- About session starts -->
+        <section class="about">
+            <h2 class="intro">
```
Comment (MeqdamAlqudah) [inline index.html:181]: - [ ] Please use the same font family as in Figma design in Figma design they used Poppins for each card title and paragraph
image

File: index.html [hunk 4]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+                   <li class="social-media"><a class="list-style" href="#" ><img src="tweeter.svg"        alt="" ></a></li>
+                   <li class="social-media"><a class="list-style" href="#" ><img src="Linkedin-icon.svg"  alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style " href="#" ><img src="mediaThree.svg"     alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFour.svg"      alt="" ></a></li> 
+                   <li class="social-media"><a class="list-style" href="#" ><img src="mediaFive.svg"      alt=""></a></li>
+    
+                </ul>
+
+                <button type="submit"  class="btnNormal">Get my resume</button>
+
```
Comment (MeqdamAlqudah) [inline index.html:205]: - [ ] Please use the same font family used in the Figma design 

| Your design | Figma design |
| ------------- | --------------- |
| image | image |

I noticed you didn't use the same font family to design the paragraph and the title of this section

File: index.html [hunk 5]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+                <ul class="lang-sec">
+                    <li class="lang-tille-link">
+                    <h4 class="lang-title">Languages</h4>
+                    <a class="" href="#">
+                    <img class="language-display-icon" src="lanDisplayIcon.svg" alt="">
+                    </a>
```
Comment (MeqdamAlqudah) [inline index.html:211]: - [ ] Please fix this weird line here and make it look like the design 

| Your design | Figma design |
| ------------- | --------------- |
| image | image |

File: index.html (hunk 4 above)
Comment (MeqdamAlqudah) [inline index.html:205]: - [ ] Please apply the hovering on the get my resume button 

image
image

File: index.html [hunk 6]
```diff
@@ -54,8 +54,204 @@ <h4 class="social-media-intro">LET’S CONNECT</h4>
...
+                    <li class="langIconDetails thd-lang">
+                    <img class="iconDetailsImg" src="cssImage.svg" alt="">
+                    <a class="langIconDetailsLink" href="#"><h4 class="lang-icon-">CSS</h4></a>   
+                     </li>
+                </ul>
+
+                <ul class="lang-sec">
+                    <li class="lang-tille-link">
+                    <h4 class="lang-title">Frameworks</h4>
+                    <a class="" href="#"><img class="language-display-icon" src="lanDisplayIcon.svg" alt=""></a>
```
Comment (MeqdamAlqudah) [inline index.html:234]: - [ ] Please align the items in the language , frameworks , sections with flex box and make sure they looks exactly like Figma design 



| Your design | Figma |
| ------------- | ------- | 
| image | image |

Please fix the arrows here and make sure that the distance between the arrow and the title is the same as in Figma design

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @henrycode460 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
# Approved 👏 

Hi there, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error at present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**